### ISSUE TITLE
Økosystemtjenester 

### ISSUE THREAD

**Issue by Caroline-A** [realfagstermer/realfagstermer#1580]
Vi har ikke noe om dette så langt jeg kan se, så jeg foreslår å legge til _[Økosystemtjenester ](https://snl.no/%C3%B8kosystemtjeneste)EN Ecosystem services_ (e.g. for [denne](https://bora.uib.no/handle/1956/16698) eller [denne](https://bibsys-almaprimo.hosted.exlibrisgroup.com/primo-explore/fulldisplay?docid=BIBSYS_ILS71497916310002201&context=L&vid=UBB&search_scope=default_scope&tab=default_tab&lang=no_NO)). 
Enten kunne det være eget emneord, eller er det noen mulige emneord den kunne falle under (**Naturressurs, Naturbruk,** Naturressursforvaltning, Miljøforvaltning) - av disse er det kanskje naturbruk som er nærmeste (?), men den er oversatt per i dag som "land management" og jeg er ikke sikkert på at denne oversettelese er faktisk riktig. Selv om ecosystem services er ofte snakket om i sammenheng med forvaltning, er de ikke det samme, så jeg ville ikke ha sammenslått med forvaltnings-termer selv.

**Comment by BioHeidi** [comment]
Ja. Har tenkt på den før. Har brukt Miljøforvaltning/Naturforvaltning/Naturressursforvaltning tidligere.
Vi kan godt opprette den og flytte over noen dokumenter!

**Comment by danmichaelo** [comment]
:heavy_check_mark:  Opprettet Økosystemtjenester (REAL032123).
